gradle/libs.versions.toml

```diff
@@ -17,6 +17,7 @@ rxjava2 = "2.2.21"
 rxandroid = "2.1.1"
 junit5 = "5.8.2"
 detekt = "1.21.0"
+detektCompose = "0.0.18"
```

Comment:
btw no need to extract version if it is not reused

Reply:
Good point